docs/EN_US/ContainerizedHPCC/ContainerizedMods/RemoteAccess.xml

```diff
@@ -0,0 +1,347 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE section PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<sect2 id="CustTutorial3" role="brk">
+  <title>Create Custom Configuration Charts for Remote File Access</title>
+
+  <para>This section shows a technique to configure two separate HPCC Systems
+  Platform environments, using Helm, where one environment (env1) exposes
+  access to its logical files via the Distributed File System (DFS) service to
+  a separate environment (env2).</para>
+
+  <para>These could be in separate Kubernetes (K8s) instances or in separate
+  namespaces in the same K8s. For simplicity, this section will use two
+  separate namespaces within the same K8s.</para>
+
+  <sect3>
+    <title>Setting up the first environment (env1)</title>
+
+    <para><orderedlist>
+        <listitem>
+          <para>Create <emphasis role="bold">env1</emphasis> namespace (if it
+          does not already exist).</para>
+
+          <para><programlisting>kubectl create ns env1</programlisting></para>
+        </listitem>
+
+        <listitem>
+          <para>Switch to the <emphasis role="bold">env1</emphasis>
+          namespace.</para>
+
+          <para><programlisting>kubectl config set-context --current --namespace=env1</programlisting></para>
+        </listitem>
+
+        <listitem>
+          <para>Add the jetstack repo and install the jetstack certificate
+          manager.</para>
+
+          <para><programlisting>helm repo add jetstack https://charts.jetstack.io
+helm repo update
+helm install cert-manager jetstack/cert-manager --version v1.7.1 --set installCRDs=true</programlisting></para>
+        </listitem>
+
+        <listitem>
+          <para>Set <emphasis role="bold">certificates.enabled</emphasis> to
+          <emphasis>true</emphasis> in a custom values yaml file.</para>
+
+          <para><programlisting>certificates:
+  <emphasis role="red">enabled: true</emphasis></programlisting></para>
+
+          <variablelist>
+            <varlistentry>
+              <term>Note:</term>
+
+              <listitem>
+                <para>This enables the use of certificates for
+                <emphasis>all</emphasis> esp services and enables mutual TLS
+                (mTLS) between components, too. If you don’t want that (for
```

Comment:
question: not sure if comma should proceed "too" ?

Reply:
To emphasize the "too" at the end of a sentence, use a comma. You can leave them out, but I prefer to use the comma.